Automattic/vip-go-mu-plugins
# Update ElasticPress to 9d48e512

## Description

Merge https://github.com/Automattic/ElasticPress/pull/143

## Changelog Description

### Plugin Updated: Enterprise Search

Bugfix: invalid values in `tax_query` resulted in broken ES queries, we've addressed this by discarding invalid values from the query.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

See the upstream PR https://github.com/10up/ElasticPress/pull/2576 for steps to verify.
